### ISSUE TITLE
Invalid certificate in user defined route shouldn't prevent router from starting

### ISSUE THREAD

**Issue by bbaptist** [flynn/flynn#4332]
Flynn version: v20170719.0

Last night I needed to do maintenance and migrate my cluster to a new instance. After bootstrap using the backup of the previous cluster I was unable to access the cluster because the router was not running. It would continuously crash with the message `lvl=eror msg="validation_error: Certificate invalid: tls: private key does not match public key" component=router`.

I verified that the environment variables for the router had a matching key and certificate. It was not clear that this error would also be triggered by user defined routes in the database. I was able to get into the postgres database for the router and set all the routes to deleted status and get the router started. Since the error message didn't inform me as to what routes were causing the issue I had to set all 150 routes to deleted and then recreate them all after I got the router up

Long story short, I think the router should just ignore user defined routes that have invalid certs and log a message stating what the route ID was that it is ignoring.

**Comment by titanous** [comment]
I'm in the process of rewriting parts of the router, including this code, so this will be fixed in the next few weeks.

**Comment by titanous** [comment]
Flynn is unmaintained and our infrastructure will shut down on June 1, 2021. See [the README](https://github.com/flynn/flynn#flynn-is-unmaintained) for details.
